rero_ils/modules/decorators.py

```diff
@@ -144,3 +146,165 @@ def decorated_view(*args, **kwargs):
             # current_app.logger.error(str(error))
             return jsonify({'message': f'{error}'}), 400
     return decorated_view
+
+
+# STATISTICS DECORATORS =======================================================
+
+def load_resource_from_route(resource, url_key, param_name=None):
```

Comment:
Seems similar to `https://github.com/inveniosoftware/invenio-records-rest/blob/b3291ba7e5472fdbd4d2bf694da9489261a477af/invenio_records_rest/views.py#L370`